Mathlib/Computability/DFA.lean

```diff
@@ -163,7 +172,80 @@ theorem pumping_lemma [Fintype σ] {x : List α} (hx : x ∈ M.accepts)
   rw [Set.mem_singleton_iff] at ha' hc'
   substs ha' hc'
   have h := M.evalFrom_of_pow hb hb'
-  rwa [mem_accepts, evalFrom_of_append, evalFrom_of_append, h, hc]
+  rwa [mem_accepts, eval, evalFrom_of_append, evalFrom_of_append, h, hc]
 #align DFA.pumping_lemma DFA.pumping_lemma
 
+section Maps
+
+variable {α' σ' : Type*} (f : α' → α) (g : σ ≃ σ')
+
+/-- Transforms the alphabet of a DFA. -/
```

Comment:
I think you can be more precise than "transforms"

Reply:
I added some more detail – LMK what you think!

```lean
/--
`M.comap f` pulls back the alphabet of `M` along `f`. In other words, it applies `f` to the input
before passing it to `M`.
-/
```

Reply:
Looks good!